mailbox/tools/quota-recompute/pom.xml

```diff
@@ -46,6 +46,10 @@
             <groupId>${james.groupId}</groupId>
             <artifactId>james-server-data-api</artifactId>
         </dependency>
+        <dependency>
+            <groupId>${james.groupId}</groupId>
+            <artifactId>james-server-data-jmap</artifactId>
+        </dependency>
```

Comment:
Sorry as in https://github.com/apache/james-project/pull/1707 this change is not acceptable to me.